.github/workflows/test_and_deploy.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           java-version: 11
 
       - name: Run tests
-        run: sbt coverage +test
+        run: sbt 'set coverageEnabled := true' +test
```

Comment:
I don't know why this change was needed, but the github action failed for scala3 without this change.